src/XIVLauncher.Common.Unix/Compatibility/CompatibilityTools.cs

```diff
@@ -12,34 +13,40 @@ namespace XIVLauncher.Common.Unix.Compatibility;
 public class CompatibilityTools
 {
     private DirectoryInfo toolDirectory;
-    private DirectoryInfo gameConfigDirectory;
 
-    private const string WINE_GE_RELEASE_URL = "https://github.com/GloriousEggroll/wine-ge-custom/releases/download/GE-Proton7-8/wine-lutris-GE-Proton7-8-x86_64.tar.xz";
-    private const string WINE_GE_RELEASE_NAME = "lutris-GE-Proton7-8-x86_64";
+    private StreamWriter logWriter;
 
-    public string Wine64Path => Path.Combine(toolDirectory.FullName, WINE_GE_RELEASE_NAME, "bin", "wine64");
-    public string WineServerPath => Path.Combine(toolDirectory.FullName, WINE_GE_RELEASE_NAME, "bin", "wineserver");
+    private const string WINE_TKG_RELEASE_URL = "https://github.com/Kron4ek/Wine-Builds/releases/download/7.6/wine-7.6-staging-tkg-amd64.tar.xz";
+    private const string WINE_TKG_RELEASE_NAME = "wine-7.6-staging-tkg-amd64";
 
-    public DirectoryInfo Prefix { get; private set; }
-    public DirectoryInfo DotnetRuntime { get; private set; }
     public bool IsToolReady { get; private set; }
 
-    public bool IsToolDownloaded => File.Exists(Wine64Path) && this.Prefix.Exists;
+    public readonly WineSettings wineSettings;
```

Comment:
This should be a property: `public WineSettings Settings { get; private set; }`
Looks good otherwise, thanks :)

Reply:
Should be fixed now